src/core_modules/capture-core/components/Pages/NewRelationship/RegisterTei/exposedHelpers/getRelationshipNewTei.js

```diff
@@ -1,6 +1,7 @@
 // @flow
 import uuid from 'd2-utilizr/src/uuid';
-import moment from 'capture-core-utils/moment/momentResolver';
+import { moment } from 'capture-core-utils/moment';
```

Comment:
looks like this is not being used (?) Can we delete it if it is not used?

Reply:
Caused by adding the default for the momentDate parameter in getFormattedStringFromMomentUsingEuropeanGlyphs function. I will keep the parameter without default so still need this here.